Assets/src/ElectricPokemon.java

```diff
@@ -0,0 +1,47 @@
+package Assets.src;
+
+public class ElectricPokemon extends Pokemon{
+
+    private String paralyze;
+    private boolean statics;
+
+    public ElectricPokemon(String name, int level, int baseAttack, int baseDefence, int baseHp, String type, String paralyze, boolean statics, String trainerName) {
+        super(name, level, baseAttack, baseDefence, baseHp, type, trainerName);
+        this.paralyze = paralyze;
+        this.statics = statics;
+    }
+
+    Pokemon electric = new Pokemon("Elektrabuzz", 45, 16, 5, 30, "Electric",
```

Comment:
Je wilt hier in je class eigenlijk geen object aan maken.

Je class dient te dienen als 'Blauwdruk'